nav2_bt_navigator/src/bt_navigator.cpp

```diff
@@ -28,7 +29,7 @@ namespace nav2_bt_navigator
 {
 
 BtNavigator::BtNavigator()
-: nav2_util::LifecycleNode("bt_navigator", "", false),
+: nav2_util::LifecycleNode("bt_navigator", "", true),
```

Comment:
Not sure why this was changed, can't think of a reason why `use_rclcpp_node` would be set to true while initializing `nav2_util::LifecycleNode`. Can you explain?

Reply:
As I mentioned in the previous comment, changing this to true makes available the `rclcpp_node_` protected member variable that I was then passing to the `OdomSmoother`.